src/dynamodb/gameStateDBService.ts

```diff
@@ -349,6 +353,54 @@ export const resetPlayedTileInteraction = async (gameId: string): Promise<GameSt
   return parseDynamoDBAttribute<GameState>(res);
 };
 
+/**
+ * Starts new game round with new hands, wall and reset game round values.
+ * @param gameId gameId
+ * @param connectionIds connectionIds of players in gameId
+ */
+export const startNewGameRound = async (gameId: string, connectionIds: string[]): Promise<GameState> => {
+  const newWall = new HongKongWall();
+
+  // Generate hand for each user
+  const hands: UserHand[] = [];
+  connectionIds.forEach((connectionId: string) => {
+    const hand = {
+      connectionId,
+      hand: newWall.generateHand(),
+    };
+    hands.push(hand);
+  });
+
+  const updateParam: DocumentClient.UpdateItemInput = {
+    TableName: GAME_STATE_TABLE,
+    Key: {
+      gameId,
+    },
+    ConditionExpression: 'attribute_exists(#gameIdKey)',
+    ExpressionAttributeNames: {
+      '#gameIdKey': 'gameId',
+      '#currentTurnKey': 'currentTurn',
+      '#wallKey': 'wall',
+      '#handsKey': 'hands',
+    },
+    ExpressionAttributeValues: {
+      ':initialCurrentTurn': 0,
+      ':initialWall': newWall,
+      ':initialHands': hands,
+    },
+    ReturnValues: 'ALL_NEW',
+    UpdateExpression: `
+      SET #currentTurnKey = :initialCurrentTurn,
+          #wallKey        = :initialWall,
+          #handsKey       = :initialHands
```

Comment:
- `currentTurn` attribute is not used at the moment, so it can be removed;
- need to reset `currentIndex` to 52 as a new wall is used in the new round (see `initGameState` function for more details)
- and maybe reset `interactionCount` and `playedTileInteractions` to make sure they are empty at their initial state
```
interactionCount: 0,
playedTileInteractions: [],
```

Reply:
done.